queue/queueservice.js

```diff
@@ -400,8 +562,31 @@ class QueueService {
     // Find name of azure queue
     var queueName = await this.ensurePendingQueue(provisionerId, workerType);
 
-    // Get queue meta-data
-    return this._countMessages(queueName);
+    // Construct queue w. legacy name
+    // TODO: Remove this once legacy queues are empty...
+    var legacyName = [
+      this.prefix,    // prefix all queues
+      base32.encode(decodeUrlSafeBase64(provisionerId))
+        .toString('utf8')
+        .toLowerCase()
+        .replace(/=*$/, ''),
+      base32.encode(decodeUrlSafeBase64(workerType))
+        .toString('utf8')
+        .toLowerCase()
+        .replace(/=*$/, ''),
+      '1'             // priority, currently just hardcoded to 1
+    ].join('-');
+
+    // Find messages count from primary and legacy queue
+    var [
+      count,
+      legacyCount
+    ] = (await Promise.all([
```

Comment:
So I know for now we're operating only on one main queue (2 while transitioning between legacy to the new naming) , but at some point I know we spoke of supporting queue priority.  Hardcoding the queues here for counts seems to make it rigid with the queues that are supported.  Shouldn't this iterate over something returned from listQueues or similar?